internal/pkg/configloader/config.go

```diff
@@ -28,6 +31,31 @@ func (t *Tool) DeepCopy() *Tool {
 	return &retTool
 }
 
+// LoadConf reads an input file as a Config struct.
+func LoadConf(fname string) *Config {
```

Comment:
These codes are duplicated in `cmd/main.go`

Reply:
cmd.main sets viper.AutomaticEnv() and binds ENV vars and flags (cmd related);

LoadConf reads the config file (config file related, hence the name "load conf")